src/main/webapp/app/exercises/programming/manage/update/update-components/programming-exercise-difficulty.component.html

```diff
@@ -60,5 +60,30 @@
                 </label>
             </div>
         }
+        <!-- only show this for non-exam exercises -->
+        @if (programmingExercise.exerciseGroup === undefined) {
```

Comment:
I think this should be sufficient (but not 100% sure)
```suggestion
        @if (!programmingExercise.exerciseGroup) {
```

Reply:
Should be the same for this case, changed 👍